mobile/test/common/integration/base_client_integration_test.cc

```diff
@@ -95,7 +70,8 @@ BaseClientIntegrationTest::BaseClientIntegrationTest(Network::Address::IpVersion
   defer_listener_finalization_ = true;
   memset(&last_stream_final_intel_, 0, sizeof(envoy_final_stream_intel));
 
-  builder_.addLogLevel(getPlatformLogLevelFromOptions());
+  builder_.setLogLevel(
+      static_cast<Logger::Logger::Levels>(TestEnvironment::getOptions().logLevel()));
```

Comment:
why is the cast required?  I thought Logger::Logger::Levels is the standard ?

Reply:
The `TestEnvironment::getOptions().logLevel()` returns `spdlog::level::level_enum`, but `spdlog::level::level_enum` is simply an alias to `Logger::Logger::Levels`.